### ISSUE TITLE
Ethers types in Waffle Chai

### ISSUE THREAD

**Issue by alcuadrado** [EthWorks/Waffle#195]
Hey @sz-piotr, 

I just saw [this comment](https://github.com/EthWorks/Waffle/blob/f62ef4a2df0d86a0f28633b650c0f034c46a8455/waffle-chai/src/types.ts#L3) of yours and would love to help with this. 

Can you expand a little on what problems you encountered?

**Comment by sz-piotr** [comment]
Yeah! What i was trying to do is have a type declaration that uses both
```ts
/// <reference types="chai" />
```
and
```ts
import { Wallet, utils } from 'ethers'
```

I was getting a lot of compiler errors and so right now this file has `any` where the types from `ethers` should be.
